packages/relayer/src/core.rs

```diff
@@ -657,7 +657,7 @@ pub(crate) async fn handle_account_transport(
     Ok(())
 }
 
-pub(crate) fn extract_account_key_from_subject(subject: &str) -> Result<String> {
+pub(crate) fn extract_account_key_from_email_address(subject: &str) -> Result<String> {
```

Comment:
Function arg name should be `email_addr`.

Reply:
Fixed